integration-tests/gradle/src/main/resources/conditional-dependencies/ext-f/runtime/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
 
 quarkusExtension {
     deploymentArtifact = "org.acme:ext-f-deployment:1.0-SNAPSHOT"
+    disableValidation = true
```

Comment:
What was the reason for disabling validation here and in other projects?

Reply:
In the test module, we have multiple extensions in a multi module project. Dependencies between module are declared using the following syntax `project(':ext-b:deployment')`. In the validation task, when looking for deployment dependencies, we look for what was declared in `quarkus-extension.properties` files. In this project this differs from `project.group:project.name` (this is due to a limitation of having multiple module calls `runtime` and `deployment`). I was thinking of pushing a commit to improve that later on.

Reply:
Does it mean that validation wouldn't work for extensions in the same project?

Reply:
if we have multiple extension in a the same project that depends to each other, the validation will fail yes.

Reply:
We probably shouldn't enable it by default then until it's supported.

Reply:
Should I remove the task from the graph ? this would avoid introducing a "breaking change" in further versions.

Reply:
Yes, I think so.

Reply:
I pushed a fix, the task must be explicitly called to be executed

Reply:
So these `disableValidation = true` could be removed now?

Reply:
sure done :)